elasticsearch_logs_loki.yml

```diff
@@ -0,0 +1,12 @@
+---
+- hosts: elasticsearch
+  become: true
+  vars_files:
+    - vars/common.yml
+
+  roles:
+    - name: push logs files to loki using promtail
+      role: promtail
+      tags: promtail
+      vars: 
+        es_logs_to_loki: true
```

Comment:
This one seems redundant with the `elasticsearch.yml` role (beyond a testing/development shortcut), it seems simple enough to just execute Ansible with `-t promtail`.

Please remove it to prevent losing oversight amongst playbooks!